Dockerfile

```diff
@@ -0,0 +1,21 @@
+# TODO: change to a slimmer and latest version
+FROM golang:1.8.5-alpine as builder
+
+# setup the working directory
+WORKDIR /go/src/github.com/hasura/gitkube
+
+# copy source code
+COPY vendor vendor
+COPY pkg pkg
+COPY cmd cmd
+
+# build the source
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /bin/gitkube cmd/gitkube-controller/main.go
```

Comment:
should we call this binary gitkube-controller?

Reply:
We can!